### ISSUE TITLE
Ошибка обновления рендера на клиенте

### ISSUE THREAD

**Issue by plusbpm** [derbyparty/derby-faq#48]
У нас недавно появилась ошибка при рендере на клиенте такого характера:
```
p.localUpdate	@	eventmodel.js:333
p.update	@	eventmodel.js:349
p._set	@	eventmodel.js:416
(anonymous function)	@	eventmodel.js:537
p._each	@	eventmodel.js:303
p._each	@	eventmodel.js:314
p._each	@	eventmodel.js:314
p._each	@	eventmodel.js:314
p.mutate	@	eventmodel.js:519
p.set	@	eventmodel.js:536
(anonymous function)	@	Page.js:151
i	@	events.js:232
i.emit	@	events.js:99
u.emit	@	events.js:77
i	@	mutators.js:41
s._mutate	@	mutators.js:15
s._set	@	mutators.js:44
s.set	@	mutators.js:31
t.exports.t.setState	@	lenta-item-post.coffee:2
t.exports.t.bootstrap	@	lenta-item-post.coffee:2
(anonymous function)	@	lenta-item-post.coffee:20
u.wrapCallback	@	events.js:44
s.add	@	util.js:54
(anonymous function)	@	browser.js:24
```

Возникает непонятно по какому принципу, уже переделал код одного компонента что бы избавиться от нее, но как только избавился в одном компоненте она появилась из другого компонента.
Подскажите в чем может быть проблема? Где искать решение ? Заранее спасибо.

**Comment by zag2art** [comment]
Вы, наверное, не последнюю версию дерби используете. Нейт вроде здесь https://github.com/derbyjs/derby/commit/9064964a84905cae0fedd6e26e586003a2451948 это фиксил.

**Comment by plusbpm** [comment]
да, используем не последнюю версию дерби, но и обновить выйдет не скоро, код приложения завязан на возможности которые уже ушли, за ссылки спасибо, посмотрю

**Comment by plusbpm** [comment]
Просто странно как то, ведь не было такой проблемы раньше, при этом дерби не обновлялся, но ошибка вдруг появилась ... шайтан

**Comment by zag2art** [comment]
Один товарищ говорит, что ошибка проявлялась только в хроме и только с определенной версии. Типа, в том же яндекс-браузере (который на том же движке) - ошибка так и не появилась.

**Comment by zag2art** [comment]
Может, действительно из-за обновлений в браузере. хз, я сам не смотрел

**Comment by plusbpm** [comment]
вполне возможно что именно из за браузера, я действительно забыл написать что ошибка проявляется только в хроме (lin,win,mac), и заметил что с начала ошибка появилась у пользователей win и mac, а у меня в lin только спустя неделю или две, но у меня и обновления проходят раз в две недели ... хоть у гугль пиши ...

**Comment by droganov** [comment]
Да, из-за браузера. Сначала в хроме, через неделю и в опере, ЯБ видимо просто не успел обновить двигло
